manual/developer/admin/README.md

```diff
@@ -0,0 +1,323 @@
+```ditaa
+                                 +---------------+
+                                 | DriverChannel |
+                                 +-------+-------+
+                                         |1
+                                         |                 topology
++-----------------+    query   +---------+---------+        events
+| TopologyMonitor +------+---->| ControlConnection +-----------------+
++-----------------+      |     +---------+---------+                 |
+         ^               |               |                           |
+         |               |               |     topology+channel      V
+   get   |     +---------+        refresh|          events      +----------+
+node info|     |                  schema |         +------------+ EventBus |
+         |     |                         |         |            +-+--------+
++--------+-----+--+                      |         |              ^      ^
+| MetadataManager |<-------+-------------+         |          node|      |
++--------+-------++        |                       |         state|      |
+         |       |         | add/remove            v        events|      |
+         |1      |         |   node     +------------------+      |      |
+   +-----+----+  |         +------------+ NodeStateManager +------+      |
+   | Metadata |  |                      +------------------+             |
+   +----------+  |                                                       |
+                 +-------------------------------------------------------+
+                               metadata changed events
+```
+
+Note: the event bus is covered in the [common infrastructure](../common/event_bus) section.
+
+### Control connection
+
+The goal of the control connection is to maintain a dedicated `DriverChannel` instance, used to:
+
+* listen for server-side protocol events:
+  * topology events (`NEW_NODE`, `REMOVED_NODE`) and status events (`UP`, `DOWN`) are published on
+    the event bus, to be processed by other components;
+  * schema events are propagated directly to the metadata manager, to trigger a refresh;
+* provide a way to query system tables. In practice, this is used by:
+  * the topology monitor, to read node information from `system.local` and `system.peers`;
+  * the metadata manager, to read schema metadata from `system_schema.*`.
+
+It has its own reconnection mechanism (if the channel goes down, a new one will be opened to another
+node in the cluster) and some logic for initialization and shutdown.
+
+Note that the control connection is really just an implementation detail of the metadata manager and
+topology monitor: if those components are overridden with custom versions that use other means to
+get their data, the driver will detect it and not initialize the control connection (at the time of
+writing, the session also references the control connection directly, but that's a bug:
+[JAVA-2473](https://datastax-oss.atlassian.net/browse/JAVA-2473)).
```

Comment:
I think we should add the comment and line number https://github.com/datastax/java-driver/pull/1339/files#diff-9e9f825f2c44fb5f74b7c95ae568a28aR53
in this jira to remove this from documentation once it will be fixed